src/app/components/SubPageCard/index.tsx

```diff
@@ -11,7 +11,7 @@ import Skeleton from '@mui/material/Skeleton'
 type StyledComponentProps = {
   featured?: boolean
   isLoadingTitle?: boolean
-  title?: string
+  title?: ReactNode
```

Comment:
Is this for next steps, or leftover?

Reply:
Actually, in the end, unrelated but useful. For a while I thought that we need to implement skeleton titles (while loading) externally, and for that, passing in JSX components as title was necessary. But later I realized that `isTitleLoaded` is supported naturally, so no need to use JSX here. But it's still useful for other situations, so I didn't undo the change.